pandas/core/generic.py

```diff
@@ -3156,6 +3156,13 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
+    __copy__ = copy
```

Comment:
yeah this is better as ``def __copy__(self):``.

also let's change the same in ``pandas/indexes/base.py``. hopefully nothing will break.